src/ToolBox/superpmi/superpmi-shared/methodcontext.cpp

```diff
@@ -2335,50 +2336,86 @@ void MethodContext::recGetReadyToRunHelper(CORINFO_RESOLVED_TOKEN* pResolvedToke
                                            CORINFO_CONST_LOOKUP*   pLookup,
                                            bool                    result)
 {
-    LogError("getReadyToRunHelper NYI");
-
-    // TODO: we need a more sophisticated mapping
-
     if (GetReadyToRunHelper == nullptr)
-        GetReadyToRunHelper = new LightWeightMap<DWORDLONG, DWORD>();
+        GetReadyToRunHelper =
+            new LightWeightMap<Agnostic_GetReadyToRunHelper_TOKENin, Agnostic_GetReadyToRunHelper_TOKENout>();
+
+    CORINFO_LOOKUP_KIND genericLookup;
+    ZeroMemory(&genericLookup, sizeof(genericLookup));
+    if (pGenericLookupKind != 0)
+    {
+        genericLookup = *pGenericLookupKind;
+    }
+    Agnostic_GetReadyToRunHelper_TOKENin  key{*pResolvedToken, genericLookup, id};
+    Agnostic_GetReadyToRunHelper_TOKENout value{*pLookup, result};
 
-    // GetReadyToRunHelper->Add((DWORDLONG)cls, result);
+    GetReadyToRunHelper->Add(key, value);
 }
-void MethodContext::dmpGetReadyToRunHelper(DWORDLONG key, DWORD value)
+
+void MethodContext::dmpGetReadyToRunHelper(Agnostic_GetReadyToRunHelper_TOKENin  key,
+                                           Agnostic_GetReadyToRunHelper_TOKENout value)
 {
-    LogError("getReadyToRunHelper NYI");
+    LogError("dmpGetReadyToRunHelper NYI");
 }
```

Comment:
If honest I want to delete all dump methods because:
1) We don't use them;
2) We don't test them;
3) They shows information in non-useful format.

I can imagine scenario when dmp methods could be useful, but not in the current variant.

To implement them I will have to write like tons of printf with all fields inside CORINFO_RESOLVED_TOKEN* and CORINFO_LOOKUP_KIND* and etc. So I prefer to keep it unimplemented for now.

In the future we will add dump functions for our types and just call them here,